source/common/network/utility.h

```diff
@@ -9,6 +9,8 @@
 #include "envoy/network/connection.h"
 #include "envoy/network/listener.h"
 
+#include "common/common/statusor.h"
```

Comment:
doesn't seem like you need this in the current state of this file. However, I suspect maybe you will because I think you've added "OrDie" calls where you really need to propagate status.

Reply:
it is needed as we use absl status code here.